cime_config/tests.py

```diff
@@ -301,7 +300,6 @@
             "SMS_Ld2.ne30pg2_r05_IcoswISC30E3r5.BGCEXP_CNTL_CNPECACNT_1850.elm-bgcexp",
             "SMS_Ld2.ne30pg2_r05_IcoswISC30E3r5.BGCEXP_CNTL_CNPRDCTC_1850.elm-bgcexp",
             "SMS_D_Ld3.T62_oQU120.CMPASO-IAF",
-            "SMS_D_Ld1.ne30pg2_r05_IcoswISC30E3r5.WCYCL1850",
```

Comment:
Duplicate of `SMS_D_Ld1.ne30pg2_r05_IcoswISC30E3r5.WCYCL1850.allactive-wcprod`.

Reply:
Not exactly a duplicate since it doesn't have the testmod but doesn't add anything.